xla/stream_executor/sycl/BUILD

```diff
@@ -37,6 +37,10 @@ cc_library(
     srcs = ["sycl_platform.cc"],
     hdrs = ["sycl_platform.h"],
     visibility = ["//visibility:public"],
+    tags = [
+        "gpu",
+        "oneapi-only",
+    ],
```

Comment:
Since this does not depend on anything GPU- or ONEAPI-specific there is no need to tag this as `gpu` or `oneapi-only`.